widgets/glade/AnacondaWidgets.xml

```diff
@@ -142,14 +142,34 @@
             </properties>
         </glade-widget-class>
 
-    <glade-widget-class title="Layout Indicator"
-			            name="AnacondaLayoutIndicator"
-			            icon-name="widget-gtk-label"
-			            generic-name="AnacondaLayoutIndicator">
-      <properties>
-        <property id="label-width" default="12" />
-      </properties>
-	</glade-widget-class>
+        <glade-widget-class title="Layout Indicator"
+                            name="AnacondaLayoutIndicator"
+                            icon-name="widget-gtk-label"
+                            generic-name="AnacondaLayoutIndicator">
+            <properties>
+                <property id="label-width" default="12" />
+            </properties>
+        </glade-widget-class>
```

Comment:
Same here, I think this change should not be part of the commit or should be mentioned in the commit message.